code-scanning/properties/checkmarx-one.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Checkmarx",
+    "creator": "Checkmarx",
+    "description": "Beat vulnerabilities with more secure code.Scan your code with Checkmarx One and see results in the GitHub code scanning.",
+    "iconName": "checkmarx",
```

Comment:
Suggestion is use a smaller icon as the text wouldn't be visible to users.

Reply:
Would this be taken?

Reply:
Reducing the size won't help for a svg. the text won't visible because of the way icon is used while suggesting:
<img width="410" alt="Screenshot 2022-11-02 at 4 41 11 PM" src="https://user-images.githubusercontent.com/25999504/199475304-e7169e39-3242-4032-b560-8b595d08047a.png">

The text would not be clearly visible to general consumer